application/controllers/collection_instrument.py

```diff
@@ -209,9 +218,9 @@ def validate_one_instrument_for_ru_specific_upload(exercise, business, session):
         bound_logger.info("Successfully validated there isn't an instrument for this ru for this exercise")
 
     @with_db_session
-    def upload_instrument_with_no_collection_exercise(self, survey_id, classifiers=None, session=None):
+    def upload_eq(self, survey_id, classifiers=None, session=None):
```

Comment:
As above, this is exclusively used for eQ, the whole with_no_collection_exercise is a red herring

Reply:
Good change 👍